.travis.yml

```diff
@@ -5,4 +5,6 @@ services:
   - postgresql
 
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "CREATE DATABASE langmatetest;" -U postgres
+  - psql -c "CREATE USER langmatetest WITH PASSWORD '123';" -U postgres
+  - psql -c "ALTER DATABASE langmatetest OWNER TO langmatetest;" -U postgres
```

Comment:
We don't really need this line

Reply:
actually we don't need this file in this PR @yosefalnajjarofficial 

Remove unrelated file @MaiUbeid